go/private/context.bzl

```diff
@@ -74,6 +74,7 @@ def _new_args(go):
       "-goarch", go.mode.goarch,
       "-cgo=" + ("0" if go.mode.pure else "1"),
   ])
+  args.add(go.tags, before_each = "-tags")
```

Comment:
env.go expects `-tags` to be a comma separated list. It can't be repeated. I think `args.add(go.tags, join_with=",")` would be correct.

Reply:
good call! thanks